### ISSUE TITLE
[CWM] Add pagination to ResultsPanel

### ISSUE THREAD

**Issue by rogup** [DigitalCommons/mykomap-monolith#55]
## Description

We currently fetch all names matching a search to populate the ResultsPanel. This will probably become a performance issue if the dataset gets too big.

The search API supports pagination so we just need to implement this in the UI, maybe using this MUI component:
https://mui.com/material-ui/react-pagination/#controlled-pagination

## Acceptance Criteria
- [ ]

**Comment by rogup** [comment]
@ColmDC No, for now, I've just truncated the list to 200 so loading isn't too slow, and the user can't see past the first 200 items unless they refine their search https://github.com/DigitalCommons/mykomap-monolith/commit/f4b295f037f2b1d6434e9776d655e3331ed652ca

**Comment by ColmDC** [comment]
Can you estimate how big a task this would be?

**Comment by rogup** [comment]
I reckon 1-1.5 days: 0.5-1 day to design and create the pagination UI component, then half a day to link it up to the API and write Redux UTs

**Comment by ColmDC** [comment]
Is this one you had started @King-Mob, but put back in Dev Ready for a reason? Is so, what was the reason?

**Comment by King-Mob** [comment]
@ColmDC reason is the final piece of rohit's review, the width of the page numbers

**Comment by ColmDC** [comment]
> reason is the final piece of rohit's review, the width of the page numbers

Sorry, I'm confused. Please create link here to review you reference.

**Comment by King-Mob** [comment]
Here's the conversation about page numbers width: https://github.com/DigitalCommons/mykomap-monolith/pull/109#discussion_r1979310825

**Comment by ColmDC** [comment]
Where can I QA this one?

**Comment by ColmDC** [comment]
> Where can I QA this one?

Can you check that this feature was pushed to somewhere I can QA it?

**Comment by King-Mob** [comment]
@ColmDC Up on dev.maps.coop

(wasn't there previously because was merged into main rather than dev)

**Comment by ColmDC** [comment]
Nice.
